recipes/cwalk/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.9.2)
```

Comment:
If this the same version as the original cmakelists.txt of the library?

Reply:
yes it is! (https://github.com/likle/cwalk/blob/master/CMakeLists.txt)